packages/trpc/server/routers/viewer.tsx

```diff
@@ -174,7 +173,7 @@ const loggedInViewerRouter = router({
       weekStart: user.weekStart,
       theme: user.theme,
       hideBranding: user.hideBranding,
-      metadata: user.metadata,
+      metadata: userMetadata.parse(user.metadata),
```

Comment:
Making sure we have the metadata available as the parsed JSON

Reply:
This could break on prod. Not all users have metadata in the correct shape.